denys_filichkin/task_15/task_15.md

```
@@ -0,0 +1,15 @@
+## Jenkins
+
+1. [task 15 GD](https://drive.google.com/drive/folders/1IxGoCp3SJAdJ992PYAHYZKAl9dDuCcxz?usp=drive_link)
+2. [Github with Jenkinsfile](https://github.com/DenysPhV/devops-df/blob/dev/Jenkinsfile)
+
+### Use the previously created virtual machine with Linux, ensure that it is powered on and accessible. Complete the following tasks related to Jenkins and pipeline creation.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)